test/factories.rb

```diff
@@ -0,0 +1,23 @@
+# This will guess the User class
+FactoryGirl.define do  factory :stach, :class => 'Stache' do
+    name "MyString"
+description "MyText"
+price 1.5
+  end
```

Comment:
`end` at 6, 2 is not aligned with `factory :stach, :class => 'Stache' do` at 2, 23 or `FactoryGirl.define do  factory :stach, :class => 'Stache' do` at 2, 0.